target/linux/x86/modules.mk

```diff
@@ -4,6 +4,23 @@
 # This is free software, licensed under the GNU General Public License v2.
 #
 
+define KernelPackage/amzn-ena
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=Elastic Network Adapter (for Amazon AWS T3)
+  DEPENDS:=@TARGET_x86_64 
+  KCONFIG:=CONFIG_ENA_ETHERNET
+  FILES:=$(LINUX_DIR)/drivers/net/ethernet/amazon/ena/ena.ko
+  AUTOLOAD:=$(call AutoLoad,12,amzn-ena)
```

Comment:
This has to match the kernel module name, so ena instead of amzn-ena.
Please try if AutoProbe works instead.

Did you test this change?

Reply:
The original commit where I enabled both in the kernel config were actually tested by the guy in the forum, the move to a module was not, I thought I grasped better how kmods worked, but it seems I didn't.
I'll ask him to test again after I addressed your last feedback.